<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>notes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recource.qrc">
    <normaloff>:/resource/images/заголовок-10.png</normaloff>:/resource/images/заголовок-10.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(120, 65, 81);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>0</y>
      <width>661</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>19</width>
      <height>19</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="icon">
      <iconset resource="recource.qrc">
       <normaloff>:/resource/images/иконки_окон-12.png</normaloff>:/resource/images/иконки_окон-12.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Заметки</string>
     </attribute>
     <widget class="QLabel" name="label9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 65, 81);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>22</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout2_4">
       <item>
        <widget class="QCheckBox" name="checkBox4">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>35</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout2_8">
       <item>
        <widget class="QCheckBox" name="checkBox8">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>11</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 65, 81);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>380</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout_7">
       <item>
        <widget class="QLineEdit" name="lineEdit7">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>430</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout_8">
       <item>
        <widget class="QLineEdit" name="lineEdit8">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>130</y>
        <width>551</width>
        <height>43</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>630</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 70px;</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout_5">
       <item>
        <widget class="QLineEdit" name="lineEdit5">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>530</y>
        <width>22</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout2_10">
       <item>
        <widget class="QCheckBox" name="checkBox10">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>22</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout2_9">
       <item>
        <widget class="QCheckBox" name="checkBox9">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>22</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout2_6">
       <item>
        <widget class="QCheckBox" name="checkBox6">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 65, 81);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>480</y>
        <width>551</width>
        <height>43</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout_9">
       <item>
        <widget class="QLineEdit" name="lineEdit9">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 65, 81);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddNote">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.5, y1:0.869318, x2:0.499, y2:1, stop:0.741573 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="recource.qrc">
        <normaloff>:/resource/images/плюс3-11.png</normaloff>:/resource/images/плюс3-11.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>22</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout2_7">
       <item>
        <widget class="QCheckBox" name="checkBox7">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>330</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout_6">
       <item>
        <widget class="QLineEdit" name="lineEdit6">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 65, 81);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>589</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.5, y1:0.869318, x2:0.499, y2:1, stop:0.741573 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
font: italic 12pt &quot;Montserrat&quot;;
color: white;</string>
      </property>
      <property name="text">
       <string>Сделано:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>230</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEdit4">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 65, 81);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>22</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout2">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>530</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 65, 81);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>22</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout2_2">
       <item>
        <widget class="QCheckBox" name="checkBox2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>22</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout2_5">
       <item>
        <widget class="QCheckBox" name="checkBox5">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 65, 81);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>22</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout2_3">
       <item>
        <widget class="QCheckBox" name="checkBox3">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 65, 81);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>551</width>
        <height>44</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_10">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>530</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout_10">
       <item>
        <widget class="QLineEdit" name="lineEdit10">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_Name">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white; 
background-color: 
rgba(255, 255, 255, 0); 
font: italic 12pt Montserrat; 
text-align: center;
</string>
      </property>
      <property name="text">
       <string>Создать новую заметку</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>49</x>
        <y>180</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit3">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>571</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(120, 65, 81);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaveInDB">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>Сохранить изменения в базу данных</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.5, y1:0.869318, x2:0.499, y2:1, stop:0.741573 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="recource.qrc">
        <normaloff>:/resource/images/icons8-save-96.png</normaloff>:/resource/images/icons8-save-96.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDeleteFromDB">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>Удалить все записи за этот день из базы данных</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.5, y1:0.869318, x2:0.499, y2:1, stop:0.741573 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="recource.qrc">
        <normaloff>:/resource/images/icons8-delete-bin-96.png</normaloff>:/resource/images/icons8-delete-bin-96.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
     </widget>
     <zorder>verticalLayoutWidget_4</zorder>
     <zorder>verticalLayoutWidget_8</zorder>
     <zorder>layoutWidget_7</zorder>
     <zorder>layoutWidget_8</zorder>
     <zorder>layoutWidget_2</zorder>
     <zorder>progressBar</zorder>
     <zorder>layoutWidget_5</zorder>
     <zorder>verticalLayoutWidget_10</zorder>
     <zorder>verticalLayoutWidget_9</zorder>
     <zorder>verticalLayoutWidget_6</zorder>
     <zorder>layoutWidget_9</zorder>
     <zorder>btnAddNote</zorder>
     <zorder>verticalLayoutWidget_7</zorder>
     <zorder>layoutWidget_6</zorder>
     <zorder>label_2</zorder>
     <zorder>layoutWidget_4</zorder>
     <zorder>label6</zorder>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>verticalLayoutWidget_2</zorder>
     <zorder>verticalLayoutWidget_5</zorder>
     <zorder>label4</zorder>
     <zorder>verticalLayoutWidget_3</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget_10</zorder>
     <zorder>label_Name</zorder>
     <zorder>layoutWidget_3</zorder>
     <zorder>label7</zorder>
     <zorder>label8</zorder>
     <zorder>label10</zorder>
     <zorder>label9</zorder>
     <zorder>label5</zorder>
     <zorder>label2</zorder>
     <zorder>label</zorder>
     <zorder>label3</zorder>
     <zorder>btnSaveInDB</zorder>
     <zorder>btnDeleteFromDB</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="icon">
      <iconset resource="recource.qrc">
       <normaloff>:/resource/images/иконки_окон-14.png</normaloff>:/resource/images/иконки_окон-14.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Календарь</string>
     </attribute>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>411</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recource.qrc"/>
 </resources>
 <connections/>
</ui>
